staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -77,6 +77,81 @@ func TestValidationExpressions(t *testing.T) {
 				"self.val7 == 1.0",
 			},
 		},
+		{name: "numeric comparisons",
+			obj:    objs(int64(5), int64(10), int64(15), float64(10.0), float64(10.0), float64(10.0), int64(10), int64(10), int64(10)),
+			schema: schemas(integerType, integerType, integerType, numberType, floatType, doubleType, numberType, floatType, doubleType),
+			valid: []string{
```

Comment:
remind me, does `valid` mean these should type-check without error, or evaluate without errors, or return true?

Reply:
All of the above.